ush/config_defaults.sh

```diff
@@ -1760,7 +1760,7 @@ DO_ENKFUPDATE="FALSE"
 #
 DO_DACYCLE="FALSE"
 DO_SURFACE_CYCLE="FALSE"
-SURFACE_CYCLE_DELAY_HRS="0"
+SURFACE_CYCLE_DELAY_HRS="1"
```

Comment:
Similarly, do we want to change this in the default config? It may affect other applications.